src/spikeinterface/core/base.py

```diff
@@ -376,11 +380,6 @@ def to_dict(
             "relative_paths": (relative_to is not None),
         }
 
-        try:
-            dump_dict["version"] = imported_module.__version__
-        except AttributeError:
-            dump_dict["version"] = "unknown"
-
```

Comment:
Is it moved elsewhere ?

Reply:
there were 2 of them, now it's at lines: https://github.com/alejoe91/spikeinterface/blob/non-recursive-relative-paths/src/spikeinterface/core/base.py#L370-L373